Modules/CalcPerform.lua

```diff
@@ -348,10 +350,19 @@ local function doActorMisc(env, actor)
 	output.RemovableFrenzyCharges = m_max(output.FrenzyCharges - output.FrenzyChargesMin, 0)
 	if modDB:Flag(nil, "UseEnduranceCharges") then
 		output.EnduranceCharges = modDB:Override(nil, "EnduranceCharges") or output.EnduranceChargesMax
+		if modDB:Flag(nil, "EnduranceChargesConvertToBrutalCharges") then
+			-- we max with possible Endurance Charge Override from Config since Brutal Charges won't have their own config entry
+			-- and are converted from Endurance Charges
+			output.BrutalCharges = m_max(output.EnduranceCharges, m_min(output.BrutalChargesMax, output.BrutalChargesMin))
+			output.EnduranceCharges = 0
+		else
+			output.EnduranceCharges = m_max(output.EnduranceCharges, m_min(output.EnduranceChargesMax, output.EnduranceChargesMin))
+			output.BrutalCharges = 0
+		end
 	else
 		output.EnduranceCharges = 0
+		output.BrutalCharges = 0
 	end
```

Comment:
I think it's cleaner if you first set the charges and then overwrite them based on the flags. Example:
```suggestion
output.EnduranceCharges = 0
output.BrutalCharges = 0
if modDB:Flag(nil, "UseEnduranceCharges") then
    output.EnduranceCharges = modDB:Override(nil, "EnduranceCharges") or output.EnduranceChargesMax
    if modDB:Flag(nil, "EnduranceChargesConvertToBrutalCharges") then
        -- we max with possible Endurance Charge Override from Config since Brutal Charges won't have their own config entry
        -- and are converted from Endurance Charges
        output.BrutalCharges = m_max(output.EnduranceCharges, m_min(output.BrutalChargesMax, output.BrutalChargesMin))
    else
        output.EnduranceCharges = m_max(output.EnduranceCharges, m_min(output.EnduranceChargesMax, output.EnduranceChargesMin))
    end
end
```